app/models/conversation.rb

```diff
@@ -2,4 +2,16 @@ class Conversation < ActiveRecord::Base
   belongs_to :topic
   belongs_to :author, class_name: 'User', foreign_key: :user_id
   has_many :messages
+
+  def self.active
```

Comment:
I like that you broke this problem down and tackled it as a Ruby problem. That shows good problem solving.

In the case of problems like this in Active Record, I'd suggest googling for ways to do this efficiently in AR / Rails.

For example, here you can probably do something like this:

```ruby
Conversation.where("updated_at >", 1.day.ago)
```

A quick Google search should point you in that direction.